code/src/java/pcgen/util/Comparators.java

```diff
@@ -211,30 +186,26 @@ public int compare(Object o1, Object o2)
 			return collator.compare(String.valueOf(o1), String.valueOf(o2));
 		}
 
-		private String getSortKey(Object obj1)
+		private static String getSortKey(Object obj1)
 		{
 			String key;
 			if (obj1 == null)
 			{
 				key = "";
 			}
-			else if (obj1 instanceof CDOMObject)
-			{
-				CDOMObject co = (CDOMObject) obj1;
-				key = co.get(StringKey.SORT_KEY);
-				if (key == null)
-				{
-					key = co.getDisplayName();
+			else {
+				if (obj1 instanceof CDOMObject) {
```

Comment:
Will point this out. The code standard is curly braces are on their own lines...

if(some such thing)
{
    some code
}

I personally think it adds more lines, but it does make readability easier.